6.7.6-community/Dockerfile

```diff
@@ -50,3 +50,4 @@ WORKDIR $SONARQUBE_HOME
 COPY run.sh $SONARQUBE_HOME/bin/
 USER sonarqube
 ENTRYPOINT ["./bin/run.sh"]
+HEALTHCHECK --start-period=30s --interval=2m --timeout=3s --retries=3 CMD curl -f http://localhost:9000/ || exit 1
```

Comment:
The web service `api/system/ping` could be used instead of loading the homepage. It checks that the web server is up.